ert/data/record/_record.py

```diff
@@ -71,7 +79,21 @@ def record_type(self) -> RecordType:
 
 
 class BlobRecord(Record):
+    """The :class:`BlobRecord` is implementation of the Record class
```

Comment:
```suggestion
    """The :class:`BlobRecord` is an implementation of the Record class
```

Reply:
In general I thought that `-ation` nouns don't need an article. We should ask Terry :smile: